contrib/offline/README.md

```diff
@@ -19,3 +23,21 @@ Step(2) can be operated with:
 ```shell
 manage-offline-container-images.sh   register
 ```
+
+## generate_list.sh
+
+This script generate download files list and images list by `roles/download/defaults/main.yml` file.
```

Comment:
s/generate/generates/
s/download files list/the list of downloaded files/
s/images list/the list of container images/  for consistency with the above section.

Reply:
ok, i completed it